### DISCUSSION TITLE
[ideas] Add reference to tar_branches in tar_deps docs

### DISCUSSION THREAD

**Discussion by lunafazio** [ropensci/targets#1087] [Ideas]
### Help

- [X] I understand and agree to <https://books.ropensci.org/targets/help.html>.

### Description

Having tried sheer intuition, search engine results and even LLM suggestions, `tar_dep` seems to be the first and only thing that comes up when trying to find a way of spotting dependencies in a target. It might be helpful to mention `tar_branches` in the "See also" block, as this is what I was actually looking for.

FYI, I'm also finding some use for `tar_visnetwork()$x$edges`. Perhaps I have completely overlooked some part of the documentation, otherwise it feels like having an output along those lines would be a useful user-facing function.

(I am aware I could pass metadata throughout the pipeline to track how certain objects were constructed, but this would result in somewhat uglier code and force rerunning potentially lengthy processes if not planned for ahead of time, which can be avoided by scanning for object names and "propagating" them in post-processing)

**Comment by wlandau** [comment]
Thanks @bmfazio, added in 5cd93a2c5c22a8cfef2f860f646b47a3cfbb8e5d.

> FYI, I'm also finding some use for tar_visnetwork()$x$edges. Perhaps I have completely overlooked some part of the documentation, otherwise it feels like having an output along those lines would be a useful user-facing function.

`tar_network()$edges` also gets you this info.

**Comment by lunafazio** [comment]
Perfect, thank you!
